scripts/tap.sh

```diff
@@ -7,6 +7,9 @@ export TAP_NAME=$(basename $INPUT_REPOSITORY)
 export TAP_PATH=$TAPS_DIR/$TAP_NAME
 export TAP_ALIAS="client_tap"
 
+# Check if INPUT_TAG exists, if not, use INPUT_TAP_REF env or develop
+[[ "$INPUT_TAG" ]] || export INPUT_TAG=${INPUT_TAP_REF:-develop};
```

Comment:
Ensure the `INPUT_TAG` evn is set.
* This really should not be needed, since it already happens in `scripts/run.sh`
* But, I figured we should add it here, incase this file is called, without `scripts/run.sh` being called